src/Popover/Popover.js

```diff
@@ -10,7 +10,6 @@ import Card from '../Card';
 
 const PopoverContent = styled(Card)`
   background: ${props => props.theme.colors.white};
-  border-radius: ${props => props.theme.radii.large};
   box-shadow: 0 4px 8px
```

Comment:
Popovers are cards with elevation2. If we add that default we can remove boxshadow from here @dgalarza

Reply:
Going to tackle this as a separate PR. Created a story here - https://www.pivotaltracker.com/story/show/164466979